.claude/scripts/pr-ownership-disclosure-go/main.go

```
@@ -0,0 +1,234 @@
+		default:
+			return rest, true
+		}
+
+		// Consume the marker and at most ONE following space or tab. A second
+		// space is content indentation and must survive, so that four of them
+		// still open an indented code block inside the container.
+		rest = rest[1:]
+		if marker == '>' {
+			rest = expandLeadingTabs(rest)
+		}
+		if strings.HasPrefix(rest, " ") {
```

golangci-lint (2.12.2)

[error] S1017: should replace this if statement with an unconditional strings.TrimPrefix

(staticcheck)